api/resources_portal/views/document_views.py

```diff
@@ -253,6 +262,7 @@ class MaterialDocumentView(DocumentViewSet):
 
     # Define ordering fields
     ordering_fields = {
+        "is_archived": "is_archived",
```

Comment:
did you try `-is_archived`?

Reply:
Hmm, that might be necessary, but the current issue I'm running into is that any materials with `is_archived` are being filtered out of the result set, not misordered.